km62/Berenchuk_Olha/laboratory1.sql

```diff
@@ -1,2 +1,134 @@
 -- LABORATORY WORK 1
 -- BY Berenchuk_Olha
+
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+видаляти та вставляти дані до таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Berenchuk
+IDENTIFIED BY koko
+DEFAULT TABLESPACE "USERS"
+TEMPORARY TABLESPACE "TEMP";
+
+ALTER USER Berenchuk QUOTA 100M ON USERS;
+GRANT DROP ANY TABLE TO Berenchuk;
+GRANT INSERT ANY TABLE TO Berenchuk;
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Людина купляє певну марку телефону.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE TABLE mobile
+(
+  name_mob varchar2(20) NOT NULL
+);
+
+ALTER TABLE mobile
+ADD CONSTRAINT name_key
+PRIMARY KEY (name_mob);
+
+CREATE TABLE price
+(
+  coust_mob NUMBER(4,2) NOT NULL
+);
+
+ALTER TABLE price
+ADD CONSTRAINT coust_key
+PRIMARY KEY (coust_mob);
+
+CREATE TABLE mobile_price
+(
+  name_mob_after varchar2(20) NOT NULL,
+  coust_mob_after NUMBER(4,2) NOT NULL,
+  colour varchar2(20)
+);
+
```

Comment:
CREATE TABLE human_what_buy_mobile(
human_id_number_fk  NUMBER(40) NOT NULL
mobile_brand_fk varchar2(20) NOT NULL
shop  varchar(40) NOT NULL
time date() NOT NULL
);